tests/datatypes/test_set.py

```diff
@@ -62,6 +62,34 @@ def test_iter(self):
             """)
 
 
+    def test__ior__(self):
+        self.assertCodeExecution("""
+        a = {'a', 'b', 'c'}
+        b = {'a', 'c', 'd'}
+        a.__ior__(b)
```

Comment:
The check of `__ior__` isn't needed - it's covered automatically by the InlineBinaryOperationTests.